sdk/keyvault/keyvault-secrets/types/src/core/clientOptions.d.ts

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
Files in the ./types directory are created during the build step. We should follow https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/keyvault/keyvault and leave them out of the repo for now.